modules/miscutil/lib/orcidutils.py

```diff
@@ -675,8 +661,8 @@ def _stub_method():
                     _stub_method()
                 except DoubledIds:
                     register_exception(subject="The paper %s contains the"
-                                       "same ArXiv id %s twice." % (recid,
-                                                                    encoded),
+                                       " same arXiv id %s twice."
```

Comment:
this was the original reason for editing -- inserting a missing space in the exception message